tpm2/tpm2.go

```diff
@@ -1187,12 +1187,18 @@ func NVUndefineSpace(rw io.ReadWriter, ownerAuth string, owner, index tpmutil.Ha
 }
 
 // NVUndefineSpaceSpecial This command allows removal of a platform-created NV Index that has TPMA_NV_POLICY_DELETE SET.
-func NVUndefineSpaceSpecial(rw io.ReadWriter, index, platformPP tpmutil.Handle) error {
-	cmd, err := tpmutil.Pack(index, platformPP)
+func NVUndefineSpaceSpecial(rw io.ReadWriter, nvIndex, sessionIndex tpmutil.Handle, pAuth []byte) error {
```

Comment:
Should `sessionIndex` be renamed to `platform` to match the spec?

Reply:
This would be not correct, because the parameter in the function call does not reflect the spec. It is the session handle which authorize the access to the nv index.

Reply:
Nevermind, changed it anyway related to the comments below :)